.github/workflows/build-docker-julia.yml

```diff
@@ -7,14 +7,13 @@ on:
       - 'main'
     tags:
       - 'v*.*.*'
-  pull_request:  
-    branches:
-      - 'main'
+  pull_request:
   workflow_dispatch:
 
 env:
   REGISTRY: hub.docker.com
   IMAGE_NAME: brownccv/icefloetracker-julia
+  TEST_TAG: brownccv/icefloetracker-julia:test
```

Comment:
The lines that use this variable are commented out. Is it needed?

Reply:
Yes, reintroduced